packages/fluentui/react-northstar/src/themes/teams/components/Datepicker/datepickerCalendarHeaderStyles.ts

```diff
@@ -2,16 +2,25 @@ import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
 import { DatepickerVariables } from './datepickerVariables';
 import { DatepickerCalendarHeaderStylesProps } from '../../../../components/Datepicker/DatepickerCalendarHeader';
 
+import { pxToRem } from '../../../../utils';
+
 export const datepickerCalendarHeaderStyles: ComponentSlotStylesPrepared<
   DatepickerCalendarHeaderStylesProps,
   DatepickerVariables
 > = {
-  root: (): ICSSInJSStyle => {
+  root: ({ variables: v }): ICSSInJSStyle => {
+    return {
+      display: 'flex',
+      alignItems: 'center',
+      paddingTop: pxToRem(5),
```

Comment:
Let's move all the pxToRem calculations to the variables, for easier customization? Also that would allow us to give some context to what those values mean :)

Reply:
Good point, moved everything :)